CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -13,6 +13,9 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 ### Dependencies
 - Compile binaries using [go 1.21.6](https://github.com/etcd-io/etcd/pull/17362)
 
+### Go
```

Comment:
You did not change this.

Reply:
Yes sorry, I just realised while going though the changes. Doing it now.